README.md

```diff
@@ -1,12 +1,47 @@
-# MongoDB StackStorm Authentication Backend
+# MongoDB authentication plugin for StackStorm Community edition
```

Comment:
I think we can get rid of "for StackStorm Community edition" and just replace it with "or StackStorm" since the same plugin can also be used with the enterprise edition if wanted / needed.